functions/index.js

```diff
@@ -1,5 +1,9 @@
 import admin from 'firebase-admin';
-import { onRequest } from 'firebase-functions/v2/https';
+import { onCall, onRequest } from 'firebase-functions/v2/https';
+import {
+  getFeature as getFeatureSource,
+  search as searchSource,
+} from './search.js';
 import update from './update.js';
 
 admin.initializeApp();
```

Comment:
**question**: The admin api is used to create clients that circumvent the storage and firestore rules. I'm not sure what you need it for here?

Reply:
I'm using it for updating the remote configs. Not sure if there's a better client or not. I was just following docs. Remote configs seem pretty new so mabe there's a better way now.